circuitbreakerpanel/circuitbreakerpanel-java-mvn/README.md

```diff
@@ -0,0 +1,38 @@
+# CircuitBreakerPanel recipe for Lagom with Java
+
+
+## Implementation details
+The circuitbreakerpanel-impl/src/main/resources/application.conf contains configuration for your circuitbreaker:
+
+This configuration is specified in detail in the [`lagom documentation`](https://www.lagomframework.com/documentation/1.4.x/java/ServiceClients.html#Circuit-Breaker-Configuration)
+
+In this sample we use the circuitBreakerPanel in [`DemoServceimpl`](src/main/java/com/lightbend/lagom/recipes/cbpanel/impl/DemoServiceImpl)
+when invoking our repository layer.
+In a real scenario this repository will be interacting with a persistence layer.
+
+
+
+## Testing the recipe
+
+To start the service:
+
+```
+mvn lagom:runAll
+```
+To push the circuitBreaker into an open state just hit the service with a path parameter which is larger than 20 
+characters in length.
+e.g
+```
+curl http://localhost:9000/random/i_am_going_home_now_because_i_am_sleepy
+```
+Hitting the service with above 3 times and the circuitbreaker goes into an open state, when this happens you will
+see the CircuitBreakerOpen exception which indicates calls are failing fast.
+
+But if you hit the service with a parameter of size one , even though the repository throws an exception, the failure
+won't add towards the circuitbreakers open state.
```

Comment:
Can you explain a little more why short names fail but don't trip the breaker?

Reply:
sorry @ignasi35 , got a busy and did not get a chance till today. I'll do the other changes but for this one do you mean explain whitelisting ?

Reply:
Maybe it could even be something really simple, like a service call that takes a `boolean fail` parameter.

Reply:
@TimMoore All valid points! Another scenario that this could be applied to would be imo whitelisting data not found exception. Do you think something like this properly explained would be helpful -
Repository will have 3 scenarios -
User Id 0-100 userName found and returned 
User id 100+ userName not found -> DataNotFound Exception thrown which will be whitelisted
User id -ve -> This we can say represents  a scenario where the database is "down "

Reply:
I think that's still going to be a little obscure... it might be OK if it's explained clearly in the read me, but something more straightforward might be better.

Reply:
I agree with @TimMoore . 

Even the current wording in `README` _just_ tells you what to do but not why.

Reply:
@TimMoore  @ignasi35  Please take a look now, I have added proper explanation to the README and slightly altered the service logic.